edb/server/bootstrap.py

```diff
@@ -2258,6 +2259,18 @@ async def _populate_misc_instance_data(
             f'{edbdef.EDGEDB_SYSTEM_DB}metadata',
             json.dumps({}),
         )
+
+    await _store_static_json_cache(
+        ctx,
+        'sql_default_fe_settings',
+        json.dumps(
+            [
+                {"name": key, "value": pg_common.setting_to_sql(key, val)}
+                for key, val in dbstate.DEFAULT_SQL_FE_SETTINGS.items()
+            ]
+        )
+    )
```

Comment:
Do we need to do anything for inplace upgrade?

Reply:
@msullivan

Reply:
Oh I missed this question; I'll follow up.

Reply:
Nothing needs to be done for inplace upgrade. instdata is versioned and so gets all the new stuff without intervention.
(Though maybe something should have been done for 6.x??)